x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/TemplateParamValidator.java

```diff
@@ -83,9 +84,8 @@ public void validate(Map<String, Object> templateParams) throws ValidationExcept
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         try (InputStream stream = new BytesArray(getSchemaPropertiesAsString()).streamInput()) {
-            builder.rawValue(stream, builder.contentType());
+            builder.rawValue(stream, XContentType.JSON);
```

Comment:
This change helped with passing all BWC test cases. Not sure if would have any other impact.

Reply:
I think this is probably OK, because I don't think we usually do XContentTypes other than JSON but I'd like to defer to @carlosdelest here

Reply:
Thank you Carlos  🙌